From 850df1a121b8415c8235cbd2395ad94098ef4ce8 Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Fri, 6 Jun 2014 17:43:45 +0200
Subject: [PATCH] Disable publish-workstation= and publish-hinfo= by default

"Publish workstation" and "Publish hinfo" export some metadata about the
machine which aren't necessary for normal operations. If used to monitor
availability of remote machines, such as in a computer lab, system
administrators could enable them remotely.

https://bugzilla.redhat.com/show_bug.cgi?id=1105647
---
 avahi-daemon/avahi-daemon.conf | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/avahi-daemon/avahi-daemon.conf b/avahi-daemon/avahi-daemon.conf
index c992842..ce7914e 100644
--- a/avahi-daemon/avahi-daemon.conf
+++ b/avahi-daemon/avahi-daemon.conf
@@ -46,8 +46,8 @@ enable-wide-area=yes
 #disable-user-service-publishing=no
 #add-service-cookie=no
 #publish-addresses=yes
-#publish-hinfo=yes
-#publish-workstation=yes
+publish-hinfo=no
+publish-workstation=no
 #publish-domain=yes
 #publish-dns-servers=192.168.50.1, 192.168.50.2
 #publish-resolv-conf-dns-servers=yes
-- 
1.9.3

